package.json

```diff
@@ -56,11 +57,11 @@
     "async": "^2.5.0",
     "base32.js": "~0.1.0",
     "cids": "~0.5.2",
-    "interface-datastore": "~0.4.0",
     "datastore-core": "~0.4.0",
     "datastore-fs": "~0.4.0",
-    "datastore-level": "~0.6.0",
+    "datastore-level": "github:ipfs/js-datastore-level#windows-interop",
```

Comment:
Change to use the actual version

Reply:
@diasdavid was not ready for a review yet.  One the travis CI builds keeps on timing out.

Reply:
Roger!